### DISCUSSION TITLE
Maximum time period for Backward-forward trajectory

### DISCUSSION THREAD

**Discussion by ankitpatel0698** [uataq/stilt#129] [Users]
Dear sir,
I was wondering how long (maximum number of hours) can we run STILT model for backward - forward trajectory simulation? i.e. if i give value of **n_hours** for one month (24*30), will the model be able to provide good results.
And please give your valuable comments on the issue i created earlier https://github.com/uataq/stilt-tutorials/issues/4#issue-853197631. I am frequently getting this error for different simulations even after re-installing the all repositories.
Thanks.

**Comment by benfasoli** [comment]
This will largely depend on the meteorological data used to drive STILT but generally, the uncertainties will increase with the simulation duration. The best method to estimate uncertainties would be to use STILT's formal transport error propagation ([Lin and Gerbig, 2005](https://agupubs.onlinelibrary.wiley.com/doi/full/10.1029/2004GL021127)) but this can be time consuming.

What are you using STILT for that requires a month long simulation? Would there be a way to shorten the simulation duration and account for the far field effects another way?

**Comment by ankitpatel0698** [comment]
We have observed some fungi at a location. For now we are trying to trace down the backward trajectory to see from how far it may have arrived at the receptor location. The backward trajectory would contain possible source regions (far away) which may have influenced it. For this far field effect we are trying one month backward and forward simulation.

**Comment by benfasoli** [comment]
STILT can run longer duration simulations (e.g. one month) but the uncertainties will largely depend on the meteorology used. Since there are many different meteorological models and configurations that can be used in STILT, quantifying these uncertainties usually requires sensitivity tests dedicated to your specific use case.

Consider setting `delt = 0` to reduce the size of the output files by dynamically choosing the output timestep.
